### PR TITLE
More copy updates

### PR DESCRIPTION
**Pull request by nikeshashar**
Updated more syntax

### CODE REVIEW

File: source/javascripts/stages/stage_2.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def code
     def instructions
       <<-INSTR
 <p>Well done!</p>
-<p>Now please make the code above work by implementing a method called `upper` that will return a given string converted to all capitals</p>
+<p>Now please make the code above work by implementing a method called `upper`. This method should accept an argument and convert the string value to be in all capitals.</p>
```
Comment (dbugsy) [inline source/javascripts/stages/stage_2.rb:29]: Are you sure you want to convert the original string? The wording is a little confusing - maybe use the original wording, but also specifying the requirement for an argument?

Comment (leoallen85) [inline source/javascripts/stages/stage_2.rb:29]: +1

**Comment by leoallen85** [comment]
:shipit: except for what dan says

File: source/javascripts/stages/stage_2.rb (hunk 1 above)
Comment (nikeshashar) [inline source/javascripts/stages/stage_2.rb:29]: It doesn't ask to convert the original string value. It asks to convert the string value. 

We're changing the word return a given as people were trying to make global variables using the old syntax so I'm thinking to test this with some students to see how they handle the wording and then make further changes. None of this is going live to unaccompanied testers. It's all controlled.

**Comment by leoallen85** [comment]
:shipit: yeah go for it can always change later based on feedback @nikeshashar

**Comment by nikeshashar** [comment]
@pitchinvasion this is will change before going live for sure. I'm trying to gauge the message that people perceive when I leave them to their own devices.

**Pull request merged**